src/overlays/actors/ovl_En_Bom_Chu/z_en_bom_chu.c

```diff
@@ -32,50 +37,539 @@ const ActorInit En_Bom_Chu_InitVars = {
     (ActorFunc)EnBomChu_Draw,
 };
 
-// static ColliderSphereInit sSphereInit = {
-static ColliderSphereInit D_808F88E0 = {
-    { COLTYPE_NONE, AT_NONE, AC_ON | AC_TYPE_PLAYER, OC1_ON | OC1_TYPE_1 | OC1_TYPE_2, OC2_TYPE_2, COLSHAPE_SPHERE, },
-    { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_ON, },
+static ColliderSphereInit sSphereInit = {
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_ON | OC1_TYPE_1 | OC1_TYPE_2,
+        OC2_TYPE_2,
+        COLSHAPE_SPHERE,
+    },
+    {
+        ELEMTYPE_UNK0,
+        { 0x00000000, 0x00, 0x00 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_ON,
+    },
     { 1, { { 0, 0, 0 }, 13 }, 100 },
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_808F890C[] = {
+static InitChainEntry sInitChain[] = {
     ICHAIN_U8(targetMode, 2, ICHAIN_CONTINUE),
-    ICHAIN_VEC3F_DIV1000(scale, 10, ICHAIN_STOP),
+    ICHAIN_VEC3F_DIV1000(scale, 1000 * BOMBCHU_SCALE, ICHAIN_STOP),
+};
+
+static EffectBlureInit2 sBlureInit = {
+    0, 0, 0, { 250, 0, 0, 250 }, { 200, 0, 0, 130 }, { 150, 0, 0, 100 }, { 100, 0, 0, 50 }, 16,
+    0, 0, 0, { 0, 0, 0, 0 },     { 0, 0, 0, 0 },
 };
 
-#endif
+static Vec3f sBlureP1Model = { 0.0f, 7.0f, -6.0f };
+
+static Vec3f sBlureP2LeftModel = { 12.0f, 0.0f, -5.0f };
+
+static Vec3f sBlureP2RightModel = { -12.0f, 0.0f, -5.0f };
```

Comment:
These look like they were inlined, and if you're going to make them static, you might as well.

Reply:
By "inlined", do you mean in the same function they're used?

Reply:
Yeah, if they're this short, static, and only used in one function, I'm pretty sure they're declared as static data in that function.